pandas/core/groupby/ops.py

```diff
@@ -837,7 +839,11 @@ def agg_series(
         # test_groupby_empty_with_category gets here with self.ngroups == 0
         #  and len(obj) > 0
 
-        if len(obj) > 0 and not isinstance(obj._values, np.ndarray):
+        if (
+            len(obj) > 0
+            and not isinstance(obj._values, np.ndarray)
+            and not isinstance(obj._values, ArrowExtensionArray)
```

Comment:
I don't think this a great long term solution. I think this needs a more general solution so that this just doesn't apply to `agg_series`

Reply:
Yes, a general solution should be in cast function to handle the pyarrow situation. I will update the pr later.